cpp/mrc/include/mrc/edge/edge_builder.hpp

```diff
@@ -176,6 +176,89 @@ struct EdgeBuilder final
         sink.set_readable_edge_handle(edge);
     }
 
+    template <typename EdgeDataTypeT, typename SourceT, typename SinkT, typename SpliceInputT, typename SpliceOutputT>
+    static void make_edge_splice(SourceT& source, SinkT& sink, SpliceInputT& splice_input, SpliceOutputT& splice_output)
```

Comment:
What is the `splice_output` here? If you go from `source->sink` to `source->splice->sink`, how are there two parameters for splice?

Reply:
**Also, the splice isn't always a single node, it can be an entire module, so we can't simplify by passing a single node and considering it the input and the output.